apps/pie-microsite/test/helpers/configuration-helper.js

```diff
@@ -0,0 +1,9 @@
+exports.getBaseUrl = () => {
```

Comment:
Would be good to add some comments to these files just to explain whats going on in each as a brief overview